src/cloud-api-adaptor/versions.yaml

```diff
@@ -45,11 +45,11 @@ git:
     reference: v1.5.0
   kbs:
     url: https://github.com/confidential-containers/trustee
-    reference: e890fc90c384207668fa3a4d6a2f2a2d652797ee
```

Comment:
Hi @stevenhorsman !

e890f is the trustee version used on kata's 3.8.0  (https://github.com/kata-containers/kata-containers/blob/3.8.0/versions.yaml#L237). Do we need a new version? that's why this update?

Reply:
Yeah sorry, this change was reverted: https://github.com/kata-containers/kata-containers/pull/10178, so I'm not sure why I picked the wrong one. I'll revert this, Thanks!